apps/core/src/components/transaction/info/UnstakeTransactionInfo.tsx

```diff
@@ -27,21 +27,21 @@ export function UnstakeTransactionInfo({
     renderExplorerLink,
 }: UnstakeTransactionInfoProps) {
     const json = event.parsedJson as {
-        principal_amount?: number;
-        reward_amount?: number;
+        principal_amount?: bigint;
+        reward_amount?: bigint;
```

Comment:
these two are not `bigint` neither `number`, they are `string`

![image](https://github.com/user-attachments/assets/92311774-60e2-4ab8-852f-f8a19bc98487)

```json
{
    "id": {
        "txDigest": "GEKf5VksPPsFuGi18esM9Hg7uHRjZo3v4H2dvpaWHdwb",
        "eventSeq": "0"
    },
    "packageId": "0x0000000000000000000000000000000000000000000000000000000000000003",
    "transactionModule": "iota_system",
    "sender": "0x5803e16d23f3efa47a3eb98af8eb70a6aef6f2c2d26503b925594d2d775087a6",
    "type": "0x3::validator::UnstakingRequestEvent",
    "parsedJson": {
        "pool_id": "0xc21fe95f73941a9e1209b4d1f52f19ed5530d25484a881c28fe022f3f0af910d",
        "principal_amount": "2000000000",
        "reward_amount": "0",
        "stake_activation_epoch": "26",
        "staker_address": "0x5803e16d23f3efa47a3eb98af8eb70a6aef6f2c2d26503b925594d2d775087a6",
        "unstaking_epoch": "25",
        "validator_address": "0xa276b4c076fff55588255630e9ee35cf0d07e8d80c78991cfd58b43b687b4206"
    },
    "bcs": "LsANhAaAWUiVWa8PpJrhynnojqEiNA4exwjmmZsa41arzFvzLsYwU2Gs1Rp2fjpyShfm9imocBPPSNJQNBqekhwpneNEs3VbDyMBQ63Pga9GxWEcGPkqkd7dVogi1vDFCL9YgwiWx9pejzCBywRxMPqs5BBjfTctLWsHXF9dBcLrrgF"
}
```